hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketSetPropertyRequest.java

```diff
@@ -206,4 +207,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       return omClientResponse;
     }
   }
+
+  public static String getRequestType() {
```

Comment:
Instead of a new method, we can use annotation.

Reply:
@bharatviswa504 Yes, I pondered about adding annotation for this. Given that we already have annotations for Cleanup tables, and BelongsToLayoutVersion, I wanted to hold off on adding more annotations. If it is ok with you, after the first round of changes, we can change this to an annotation.

Reply:
The problem currently with an annotation approach is the fact that we can not use expressions like "SetAcl.name() + "-" + ObjectType.KEY" in annotations, as that does not qualify as a constant.

Reply:
If that is the problem we can define a static variable with direct value and do it :)

Reply:
@bharatviswa504 As discussed offline, we can revisit in the next iteration.